Documentation/guides/BuildProcess.md

```diff
@@ -86,63 +85,65 @@ MSBuild items, properties and targets.
 
 The following build targets are defined for Xamarin.Android projects:
 
--   **Build** &ndash; Builds the package.
+- **Build** &ndash; Builds the package.
+
+- **BuildAndStartAotProfiling** &ndash; Builds the app with an
+  embedded AOT profiler, sets the profiler TCP port to
+  `$(AndroidAotProfilerPort)`, and starts the default activity.
+
+  The default TCP port is `9999`.
+
+  Added in Xamarin.Android 10.2.
+
+- **Clean** &ndash; Removes all files generated by the build process.
+
+- **FinishAotProfiling** &ndash; Collects the AOT profiler data from
+  the device or emulator through the TCP port
+  `$(AndroidAotProfilerPort)` and writes them to
+  `$(AndroidAotCustomProfilePath)`.
 
--   **Clean** &ndash; Removes all files generated by the build process.
+  The default values for port and custom profile are `9999` and
+  `custom.aprof`.
 
--   **Install** &ndash; Installs the package onto the default device or
-    virtual device.
+  To pass additional options to `aprofutil`, set them in the
+  `$(AProfUtilExtraOptions)` property.
 
--   **Uninstall** &ndash; Uninstalls the package from the default
-    device or virtual device.
+  This is equivalent to:
 
--   **SignAndroidPackage** &ndash; Creates and signs the package
-    (`.apk`). Use with `/p:Configuration=Release` to generate
-    self-contained "Release" packages.
+  ```
+  aprofutil $(AProfUtilExtraOptions) -s -v -f -p $(AndroidAotProfilerPort) -o "$(AndroidAotCustomProfilePath)"
+  ```
 
--   **StartAndroidActivity** &ndash; Starts *launch* activity on the
-    device or the running emulator. The launch activity can be
-    overriden by the `$(AndroidLaunchActivity)` property.
-    
-    Added in Xamarin.Android v10.3.
-    
-    This is equivalent to `adb shell am start @PACKAGE_NAME@/$(AndroidLaunchActivity)`.
+  Added in Xamarin.Android 10.2.
 
--   **StopAndroidPackage** &ndash; Completely stops the application
-    package on the device or the running emulator.
-    
-    Added in Xamarin.Android v10.3.
-    
-    This is equivalent to `adb shell am force-stop @PACKAGE_NAME@`.
+- **Install** &ndash; Installs the package onto the default device or
+  virtual device.
 
--   **UpdateAndroidResources** &ndash; Updates the
-    `Resource.designer.cs` file. This target is usually called by the
-    IDE when new resources are added to the project.
+- **Uninstall** &ndash; Uninstalls the package from the default
+  device or virtual device.
 
--   **BuildAndStartAotProfiling** &ndash; Builds the package with
-    embedded AOT profiler, sets the AOT profiler socket port to
-    `$(AndroidAotProfilerPort)` and starts the *launch* activity.
+- **SignAndroidPackage** &ndash; Creates and signs the package
+  (`.apk`). Use with `/p:Configuration=Release` to generate
+  self-contained "Release" packages.
```

Comment:
I hadn't noticed before that these were out of alphabetical order (dating back before the last sync). I can add a small additional commit to this PR to alphabetize them for the sake of simplicity and consistency.